infra/docker-evm-blockhain/start.sh

```diff
@@ -0,0 +1,128 @@
+#!/bin/bash
+
+# Set log file
+LOG_FILE="debug.log"
+
+# Clear log file content if it exists
+> $LOG_FILE
+
+# Set network ID, block time, and ports
+NETWORK_ID=14578
```

Comment:
Are there any semantics for 14578? or just some random number? if so, why not a "nice" number like 10000? (Just to understand the meaning of these items)

Reply:
The number `14578` used for the `NETWORK_ID` is just an arbitrary number chosen for this example. There isn't any specific semantic associated with it. In a real-world scenario, we would choose a unique network ID. 
We could indeed use a number like 10000 or any other number we prefer.